[2022-12-04 11:00:05,766] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: dag1.local_to_gcs_task scheduled__2022-11-03T13:50:00+00:00 [queued]>
[2022-12-04 11:00:05,822] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: dag1.local_to_gcs_task scheduled__2022-11-03T13:50:00+00:00 [queued]>
[2022-12-04 11:00:05,823] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-12-04 11:00:05,830] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-12-04 11:00:05,831] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-12-04 11:00:05,888] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2022-11-03 13:50:00+00:00
[2022-12-04 11:00:05,903] {standard_task_runner.py:52} INFO - Started process 11390 to run task
[2022-12-04 11:00:05,916] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'dag1', 'local_to_gcs_task', 'scheduled__2022-11-03T13:50:00+00:00', '--job-id', '4879', '--raw', '--subdir', 'DAGS_FOLDER/AirflowClean2.py', '--cfg-path', '/tmp/tmpqjwsznry', '--error-file', '/tmp/tmph5ehdrif']
[2022-12-04 11:00:05,951] {standard_task_runner.py:77} INFO - Job 4879: Subtask local_to_gcs_task
[2022-12-04 11:00:06,998] {logging_mixin.py:109} INFO - Running <TaskInstance: dag1.local_to_gcs_task scheduled__2022-11-03T13:50:00+00:00 [running]> on host 9e44869ff3ef
[2022-12-04 11:00:07,331] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Fadlil
AIRFLOW_CTX_DAG_ID=dag1
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2022-11-03T13:50:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-11-03T13:50:00+00:00
[2022-12-04 11:00:09,537] {python.py:175} INFO - Done. Returned value was: None
[2022-12-04 11:00:09,886] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=dag1, task_id=local_to_gcs_task, execution_date=20221103T135000, start_date=20221204T110005, end_date=20221204T110009
[2022-12-04 11:00:10,008] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-12-04 11:00:10,136] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
